lib/classes/CLI.js

```diff
@@ -330,7 +330,7 @@ class CLI {
 
   getVersionNumber() {
     this.consoleLog(
-      `${version} (Enterprise Plugin: ${enterpriseVersion}, Platform SDK: ${sdkVersion})`
+      `Framework Core: ${version}\nPlugin: ${enterpriseVersion}\nSDK: ${sdkVersion}\n`
```

Comment:
Hmm.. seems a bit to ambiguous now thb... what about `Dashboard Plugin` and `Dashboard SDK`?

Reply:
I think the preference is to leave it somewhat ambiguous, in order to keep things more future-proof.

Reply:
I agree here 🤔

What if we use the npm package names?

```
serverless: 1.49.0
@serverless/enterprise-plugin: 1.2.3
@serverless/platform-sdk: 1.2.3
```